read-the-docs-2020-stats.rst

```diff
@@ -0,0 +1,117 @@
+.. post:: Jan 14, 2021
+   :tags: stats, year-in-review
+
+Read the Docs 2020 Stats
+========================
+
+2020 was a rough year for everyone,
+including our team.
+We managed to make it through,
+and continue to have 5 folks working full-time to make Read the Docs better for you.
+
+We are going into 2021 with a :doc:`new grant </czi-grant-announcement>`,
+which will require us to do some hiring.
+We also launched our `EthicalAds network`_,
+which is bringing our approach to sustainability to the tech community as a whole.
+
+As we have for the past 7 years,
+we are proud to share our 2020 stats post.
+
+.. _EthicalAds network: https://www.ethicalads.io/
+
+.. note::
+
+	You can always see our stats for the last `30 days`_.
+
+	Our posts from 2013_, 2014_, 2015_, 2016_, 2017_, 2018_, and 2019_ are also available.
+
+.. _Read the Docs: https://readthedocs.org/
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+.. _2016: https://blog.readthedocs.com/read-the-docs-2016-stats/
+.. _2017: https://blog.readthedocs.com/read-the-docs-2017-stats/
+.. _2018: https://blog.readthedocs.com/read-the-docs-2018-stats/
+.. _2019: https://blog.readthedocs.com/read-the-docs-2019-stats/
+
+
+Page Views
+----------
+
+Our stats:
+
+* 580 Million Page Views (from 465 million)
+* 156 Million Unique Visitors (from 120 million)
+
+.. From Google Analytics
+
+These numbers are not completely accurate, since we :doc:`implemented Do Not Track support </do-not-track>` our analytics pageview figures understate our actual traffic.
+We estimate that around 6% of users are not counted here.
+We also don't count all the users who block Google Analytics with a browser extension.
+
+Site Stats
+----------
+
+The stats, in total numbers:
+
+* 240,000 projects (from 200,000)
+* 430,000 users (from 400,000)
+* 2,480,063 builds (from 10 million total last year)
```

Comment:
previously we weren't counting builds from versions that were deleted :), and also, we don't count builds from projects that were deleted. Not sure if that is worth mentioning.

Reply:
@stsewd yea, good point. 

@humitos that sounds...expensive to calculate. I think the total number of builds is probably fine, unless we have a reason to include it?

Reply:
I think it's a more accurate measure of platform usage, together with pageviews, than number builds now --in particular if we delete them.

Reply:
Thinking more, the number of builds counts the different in the primary key, so it should be valid for the number of builds in the year.

Reply:
Good point!